### PR TITLE
docs(react-messagebar): rework examples to improve docs

### PR DESCRIPTION
**Pull request by layershifter**
## New Behavior

Fixes typos, simplifies examples and cleanups code in them.

### CODE REVIEW

**Review by ling1726 — APPROVED** [review]
Please add a changefile for the typos in shipped code

File: packages/react-components/react-message-bar-preview/stories/MessageBar/Default.stories.tsx [hunk 1]
```diff
@@ -1,17 +1,12 @@
...
+import { Link } from '@fluentui/react-components';
+import { MessageBar, MessageBarTitle, MessageBarBody } from '@fluentui/react-message-bar-preview';
 
 export const Default = () => (
   <MessageBar>
     <MessageBarBody>
       <MessageBarTitle>Descriptive title</MessageBarTitle>
       Message providing information to the user with actionable insights. <Link>Link</Link>
     </MessageBarBody>
-    <MessageBarActions containerAction={<Button appearance="transparent" icon={<DismissRegular />} />}>
```
Comment (ling1726) [inline packages/react-components/react-message-bar-preview/stories/MessageBar/Default.stories.tsx:12]: nit: The intention in the default example is to showcase all the slots, it's not a requirement for me but rather a nice to have

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/60a911e66db784521a35c1e1ac0362a3f2d8169f'>60a911e66db784521a35c1e1ac0362a3f2d8169f</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Pull request merged**